spec/controllers/presentations_controller_spec.rb

```diff
@@ -164,6 +198,18 @@
         end
       end
     end
+
+    describe 'PATCH #update' do
```

Comment:
Generally we have 1 describe block for each methods and then nest the contexts below that. Here we have the reverse, The contexts outside the describe blocks